### PR TITLE
OWL-Time - Editorial changes in response to #1114

### PR DESCRIPTION
**Pull request by dr-shorthair**
Make editorial changes requested by Roger Lott in the course of OGC adoption (documented in #1114 )

For preview see https://rawgit.com/w3c/sdw/time-issue1114/time/

### CODE REVIEW

File: time/errata.html [hunk 1]
```diff
@@ -41,8 +41,14 @@ <h2 id="known-errors">Known errata in Time Ontology in OWL</h2>
...
           <dd>Description: As explained in the text, the domain of the <code>hasTemporalDuration</code> property is <code>TemporalEntity</code>. However, the <code>hasTemporalDuration</code> property does not appear in the model diagram for the <code>TemporalEntity</code> class.</dd>
           <dd>Correction: Add <code>hasTemporalDuration : TemporalDuration</code> to the model of the <code>TemporalEntity</code> class in Figure 1.</dd>
+          <dt><a id="entry-2">2. Alignment with OGC terminology for Temporal Coordinate Reference System</dt>
+          <dd>Added: 28 Feb 2019</dd>
+          <dd>Type: Editorial</dd>
+          <dd>Refers to: <a href="https://www.w3.org/TR/owl-time/#trs-clock-calendar">3.2 Temporal reference systems, clocks, calendarss</a> and <a href="https://www.w3.org/TR/owl-time/#time:TRS">4.1.18 Temporal reference system</a></dd>
+          <dd>Description: The terminology "Temporal Coordinate System" taken from from ISO 19108 has been superseded by "Temporal Coordinate Reference System" in ISO 19111:2019 (also published as OGC Abstract Specification - Topic 2)</dd>
```
Comment (tidoust) [inline time/errata.html:48]: s/from from/from

**Review by tidoust — COMMENTED** [review]
Any reason why the index.html appears completely modified? Any way to restrict the changes to the actual parts that you updated?

File: time/errata.html (hunk 1 above)
Comment (dr-shorthair) [reply time/errata.html:48]: Oh dear. No I have no idea ... is it some kind of whitespace normalization?

**Comment by dr-shorthair** [comment]
Replaced by w3c/sdw#1116 
in which only the intended changes are diffs.

**Pull request closed without merging**